File: scala/specs/java_maven/src/main/java/fits/sample/Comment.java
package fits.sample;

import java.util.Date;

public class Comment {
	private String content;
	private Date createdDate;

	public Comment() {
		this.createdDate = new Date();
	}

	public String getContent() {
		return this.content;
	}

	public Date getCreatedDate() {
		return this.createdDate;
	}
}

File: blog/20110521/scalatra-scalate-morphia/src/main/scala/models/Comment.scala
package fits.sample.models

import java.util.Date
import com.google.code.morphia.annotations._

class Comment {

	def this(content: String, user: User) = {
		this()
		this.content = content
		this.user = user
	}

	var content: String = ""
	var createdDate: Date = new Date()

	@Reference var user: User = null

}
